android/libraries/rib-badoo/src/main/java/com/badoo/common/rib/routing/action/RoutingAction.kt

```diff
@@ -0,0 +1,23 @@
+package com.badoo.common.rib.routing.action
+
+import com.badoo.common.rib.BaseViewRouter
+import com.uber.rib.core.RibView
+
+interface RoutingAction<V : RibView> {
+
+    fun onExecuteCreateTheseRibs() : List<() -> BaseViewRouter<*, *>> =
+        emptyList()
+
+    fun onExecute() {
```

Comment:
I still believe, that attach/detach semantics sound better and more consistent

Reply:
A RIB can be attached / detached. A routing action can be executed, it's not attached or detached. The problematic part is how to name the method called on the routing action it when you leave a configuration.